lit/src/shared/lost_at_sea/utils.ts

```diff
@@ -150,25 +194,30 @@ export function getRatingQuestionFromPair(
 }
 
 /**
- * Get Lost at Sea item item pair ranking answer.
+ * Check if stage is part of the LostAtSea game.
  */
-export function getLostAtSeaPairAnswer(item1: ItemName, item2: ItemName) {
-  const ranking1 = getLostAtSeaItemRanking(item1);
-  const ranking2 = getLostAtSeaItemRanking(item2);
-
-  return (ranking1 < ranking2) ? item1 : item2;
+export function isLostAtSeaGameStage(stage: StageConfig) {
+  return stage.game === LAS_ID;
 }
 
 /**
- * Get Lost at Sea item ranking.
+ * Check if stage is a LostAtSea rating survey
  */
-export function getLostAtSeaItemRanking(item: ItemName) {
-  return ITEMS[item].ranking;
+export function isLostAtSeaSurvey(stage: StageConfig) {
```

Comment:
Maybe

return (isLostAtSeaGameStage) and (stage.kind is survey) ?

Otherwise, rename since this might not be the logic that's checked here